faqs/galaxy/galaxy_creating_an_account.md

```diff
@@ -0,0 +1,26 @@
+---
+title: How to create an account on UseGalaxy.org?
+area: UseGalaxy.org
+layout: faq
+box_type: tip
+contributors: [bernandez,jennaj]
+---
+
+Registering for a Galaxy account
```

Comment:
I think that you don't need this title, it's already described in the metadata what the FAQ is about